python/eos/analysis.py

```diff
@@ -621,9 +621,9 @@ def log_likelihood(self, p, *args):
         try:
             return(self._log_likelihood.evaluate())
         except RuntimeError as e:
-            eos.error('encountered run time error ({e}) when evaluating log(posterior) in parameter point:'.format(e=e))
+            eos.error(f'encountered run time error ({e}) when evaluating log(posterior) in parameter point:')
             for p in self.varied_parameters:
-                eos.error(' - {n}: {v}'.format(n=p.name(), v=p.evaluate()))
+                eos.error(f' - {p.name()}: {p.evaluate()}')
```

Comment:
All of the changes above are due to ``pyupgrade`` and should be moved to a separate commit.